email-notifier/src/test/kotlin/org/ostelco/prime/notifications/email/MandrillClientTest.kt

```diff
@@ -31,7 +31,7 @@ data class TestConfig(val modules: List<PrimeModule>): Configuration()
 
 class MandrillClientTest {
 
-    @EnabledIfEnvironmentVariable(named = "MANDRILL_API_KEY", matches = "*")
+    @EnabledIfEnvironmentVariable(named = "MANDRILL_API_KEY", matches = ".*")
```

Comment:
This is a somewhat unusual annotation, so there should be an accompanying comment stating that the test should only be run I the variable MANDRILL_API_KEY is set to anything non-null